app/app.go

```diff
@@ -0,0 +1,5 @@
+package app
+
+type App interface {
+	Name() string
```

Comment:
Please, add godoc.

Reply:
Thanks for catching this @daniilstrata, I pushed a commit to address it.